app/core/utils.py

```
@@ -1,3 +1,31 @@
+        str: The modified DATABASE_URL with asyncpg driver.
+
+    Raises:
+        ValueError: If the DATABASE_URL scheme is unsupported.
+    """
+    db_url = settings.DATABASE_URL
+    if db_url.startswith("postgresql://"):
+        db_url = db_url.replace("postgresql://", "postgresql+asyncpg://", 1)
+    elif db_url.startswith("postgres://"):
+        db_url = db_url.replace("postgres://", "postgresql+asyncpg://", 1)
+    elif not db_url.startswith("postgresql+asyncpg://"):
+        raise ValueError(f"Unsupported DATABASE_URL scheme: {db_url}")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)